crate/cli/Cargo.toml

```
@@ -21,7 +21,8 @@ staging = []
 fips = ["cosmian_kmip/fips"]
 
 [dependencies]
-actix-web = { workspace = true }
+## Avoid pulling openssl
+actix-web = { version = "4.5.1", features = ["rustls"] }
```

we can keep using workspace dep here:
```suggestion
actix-web = { workspace = true, features = ["rustls"] }
```
and update the one in top-level `Cargo.toml`